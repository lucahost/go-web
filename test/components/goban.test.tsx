import { render } from '@testing-library/react'
import Goban from '../../src/components/goban'

describe('ButtonGroup', () => {
    it('renders correctly', () => {
        const { container } = render(<Goban size={9} />)
        expect(container).toMatchSnapshot()
    })
})
