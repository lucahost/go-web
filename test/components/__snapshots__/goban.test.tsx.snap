// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonGroup renders correctly 1`] = `
<div>
  <div
    class="goban__TileRow-c65plc-0 YcwaV"
  >
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_ul.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_u.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_u.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_u.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_u.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_u.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_u.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_u.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_ur.svg"
    />
  </div>
  <div
    class="goban__TileRow-c65plc-0 YcwaV"
  >
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_l.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_r.svg"
    />
  </div>
  <div
    class="goban__TileRow-c65plc-0 YcwaV"
  >
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_l.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_-.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_-.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_r.svg"
    />
  </div>
  <div
    class="goban__TileRow-c65plc-0 YcwaV"
  >
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_l.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_r.svg"
    />
  </div>
  <div
    class="goban__TileRow-c65plc-0 YcwaV"
  >
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_l.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_-.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_r.svg"
    />
  </div>
  <div
    class="goban__TileRow-c65plc-0 YcwaV"
  >
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_l.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_r.svg"
    />
  </div>
  <div
    class="goban__TileRow-c65plc-0 YcwaV"
  >
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_l.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_-.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_-.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_r.svg"
    />
  </div>
  <div
    class="goban__TileRow-c65plc-0 YcwaV"
  >
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_l.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_m.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_r.svg"
    />
  </div>
  <div
    class="goban__TileRow-c65plc-0 YcwaV"
  >
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_dl.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_d.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_d.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_d.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_d.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_d.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_d.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_d.svg"
    />
    <img
      class="tile__TileContainer-sc-34hp8n-0 QdBAb"
      src="/Go_dr.svg"
    />
  </div>
</div>
`;
